pommerman/cli/run_battle.py

```diff
@@ -119,7 +72,7 @@ def _run(seed, record_dir=None):
         print("Game Time: ", times[-1])
 
     env.close()
-    atexit.register(functools.partial(clean_up_agents, _agents))
+    atexit.register(functools.partial(clean_up_agents, env._agents))
```

Comment:
makes more sense at this point to register an environment clean_up func that shuts down the agents as well.